hnn_core/dipole.py

```diff
@@ -101,12 +101,10 @@ def average_dipoles(dpls):
         A new dipole object with each component of `dpl.data` representing the
         average over the same components in the input list
     """
-    # need at least one Dipole to get times
-    if len(dpls) < 2:
-        raise ValueError("Need at least two dipole object to compute an"
-                         " average")
-
     for dpl_idx, dpl in enumerate(dpls):
+        if not dpls:
```

Comment:
this is not a type check. If I give `dpls = True`, then this check will not raise any error. You need to use `isinstance` ... and for `dpl` not `dpls`

Reply:
Oops I'll take the heat for this, I think this was a suggestion I gave in a brief meeting with @kenloi. Misunderstood the purpose of the original check, `isinstance()` is definitely the way to go.

Alternatively, check out how [`_validate_type()`](https://github.com/jonescompneurolab/hnn-core/blob/9b95ae83918cf91352f7b342e81fd19e22ffdcc3/hnn_core/network.py#L257l) is used in the code. It provides a boilerplate error message as these sorts of checks come up frequently.

Reply:
@jasmainak That makes sense. I'll take care of it asap. 
@ntolley Thanks for the suggestion, will check it out.